src/vm/eventpipe.cpp

```diff
@@ -193,19 +213,27 @@ void EventPipe::Shutdown()
         NOTHROW;
         GC_TRIGGERS;
         MODE_ANY;
+        // These 3 pointers are initialized once on EventPipe::Initialize
+        // FIXME: Why is shutdown called more than once?
```

Comment:
There are 17 FIXMEs and 15 TODOs in the comments of this PR. Before checking in it would be good to triage these and either remove + fix them, remove + file an issue to fix them later, or decide that they can be removed with no further action taken.

Reply:
Resolving this as I opened https://github.com/dotnet/coreclr/issues/24753 to track all FIXME/TODO.